.github/workflows/schema-updater.yaml

```diff
@@ -9,6 +9,9 @@ jobs:
       - uses: actions/checkout@v2
       - run: |
           curl https://raw.githubusercontent.com/aws-cloudformation/aws-cloudformation-resource-schema/master/src/main/resources/schema/provider.definition.schema.v1.json > src/rpdk/core/data/schema/provider.definition.schema.v1.json
+          curl https://raw.githubusercontent.com/aws-cloudformation/aws-cloudformation-resource-schema/master/src/main/resources/schema/provider.configuration.definition.schema.v1.json > src/rpdk/core/data/schema/provider.configuration.definition.schema.v1.json
```

Comment:
or https://schema.cloudformation.us-east-1.amazonaws.com/provider.definition.schema.v1.json  I think we should figure out what the right spot is